src/elm/Api.elm

```diff
@@ -5,6 +5,7 @@ module Api exposing
     , communityInvite
     , get
     , getBalances
+    , getExpiryOpts
     , getTableRows
     , signInInvitation
```

Comment:
Yeah, I was actually looking at [this post](https://jfmengels.net/simplify/#automatic-fixes-for-unused-dependencies) by the creator of elm-review, and was thinking we could maybe use some more elm-review rules (it should be on a different issue though). You can see for yourself by running

```sh
npx elm-review --template jfmengels/elm-review-unused/example
```

Well... I get 716 errors in 107 files, just of unused stuff. As you have already mentioned, we have a lot of old code, and I think this would really help with that, what do you think?

Reply:
for sure bro, the frontend does need a lot of clean up! this can help a lot!